app/views/_includes/local-header.nunjucks

```diff
@@ -8,13 +8,13 @@
 
   <h1 class="local-header__title">{{ meta.parent.title if guide else title }}</h1>
 
-  {% if localHeader and localHeader.length > 0 or guide %}
+  {% if content.header and content.header.length > 0 or guide %}
```

Comment:
Could just check for `content.header.length`. Nunjucks will do nothing (be falsey) if parent of property doesn't exist.